salt/flink/init.sls

```diff
@@ -0,0 +1,95 @@
+{% set flavor_cfg = pillar['pnda_flavor']['states'][sls] %}
+
+{% set packages_server = pillar['packages_server']['base_uri'] %}
+{% set flink_version = pillar['flink']['release_version'] %}
```

Comment:
could you rebase this branch with develop in order to get the flink_version from the pillar please

Reply:
Done.